app/models/tweet.rb

```diff
@@ -1,9 +1,14 @@
 # frozen_string_literal: true
 
 class Tweet < ApplicationRecord
-  validates :user, presence: true
+  validates :user_id, presence: true
```

Comment:
実際に該当するuserが存在するかも見てくれるように、以下のような記述にした方が良いと思います。
```suggestion
  validates :user, presence: true
```